$NetBSD$

Avoid installation of LAPACK CMake and pkg-config files when not installing
LAPACK.

--- CMakeLists.txt.orig	2019-03-23 09:03:24.021091079 +0000
+++ CMakeLists.txt
@@ -332,10 +332,12 @@ include(CPack)
 
 if(NOT BLAS_FOUND)
   set(ALL_TARGETS ${ALL_TARGETS} blas)
+  set(BLAS ON)
 endif()
 
 if(NOT LATESTLAPACK_FOUND)
   set(ALL_TARGETS ${ALL_TARGETS} lapack)
+  set(LAPACK ON)
 endif()
 
 if(BUILD_TESTING OR LAPACKE_WITH_TMG)
@@ -378,11 +380,13 @@ configure_file(${LAPACK_SOURCE_DIR}/CMAK
   ${LAPACK_BINARY_DIR}/lapack-config.cmake @ONLY)
 
 
-configure_file(${CMAKE_CURRENT_SOURCE_DIR}/lapack.pc.in ${CMAKE_CURRENT_BINARY_DIR}/lapack.pc @ONLY)
-  install(FILES
-  ${CMAKE_CURRENT_BINARY_DIR}/lapack.pc
-  DESTINATION ${PKG_CONFIG_DIR}
-  )
+if(LAPACK)
+  configure_file(${CMAKE_CURRENT_SOURCE_DIR}/lapack.pc.in ${CMAKE_CURRENT_BINARY_DIR}/lapack.pc @ONLY)
+    install(FILES
+    ${CMAKE_CURRENT_BINARY_DIR}/lapack.pc
+    DESTINATION ${PKG_CONFIG_DIR}
+    )
+endif()
 
 configure_file(${LAPACK_SOURCE_DIR}/CMAKE/lapack-config-install.cmake.in
   ${LAPACK_BINARY_DIR}/CMakeFiles/lapack-config.cmake @ONLY)
@@ -394,8 +398,10 @@ write_basic_package_version_file(
   COMPATIBILITY SameMajorVersion
   )
 
-install(FILES
-  ${LAPACK_BINARY_DIR}/CMakeFiles/lapack-config.cmake
-  ${LAPACK_BINARY_DIR}/lapack-config-version.cmake
-  DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/lapack-${LAPACK_VERSION}
-  )
+if(LAPACK)
+  install(FILES
+    ${LAPACK_BINARY_DIR}/CMakeFiles/lapack-config.cmake
+    ${LAPACK_BINARY_DIR}/lapack-config-version.cmake
+    DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/lapack-${LAPACK_VERSION}
+    )
+endif()
